integration_tests/suite/test_database.py

```diff
@@ -146,3 +148,42 @@ def on_subscription(service, _):
             }
         )
         assert_that(tracker, has_entries({'service1': True, 'service2': True}))
+
+    def test_purger(self):
+        subscription_uuid = str(uuid.uuid4())
+
+        service = SubscriptionService({'db_uri': self.db_uri})
+        service.create(
+            {
+                'uuid': subscription_uuid,
+                'name': 'test',
+                'owner_tenant_uuid': str(uuid.uuid4()),
+            }
+        )
+
+        def add_log(days_ago):
+            started_at = datetime.datetime.now() - datetime.timedelta(days=days_ago)
+            service.create_hook_log(
+                str(uuid.uuid4()),
+                subscription_uuid,
+                "failure",
+                1,
+                3,
+                started_at,
+                started_at + datetime.timedelta(minutes=1),
+                {},
+                {},
+            )
+
+        for i in range(5):
+            add_log(i)
+            add_log(i)
+
+        logs = service.get_logs(subscription_uuid)
+        assert_that(len(logs), 10)
+
+        with service.rw_session() as session:
+            SubscriptionLogsPurger().purge(2, session)
```

Comment:
be more explicit on magic number. ex:
```suggestion
            days_to_purge = 2
            SubscriptionLogsPurger().purge(days_to_purge, session)
```